app/src/main/java/com/nsoft/github/domain/usecase/NetworkingUseCase.kt

```diff
@@ -126,6 +126,54 @@ abstract class NetworkingUseCase<
                                     callParams
                                 ).execute()
                             }
+
+                            ApiCallType.PATH -> {
+                                // Ughhhhh this hack .....
+                                // The map doesn't preserve ordering. So, hopefully we get lucky by
+                                // taking the e.g. query map's values and hope they turn out alright
+                                //TODO sort them lexicographically then invert if this doesn't work out though ...
```

Comment:
and once the comment below is done, this can be deleted since it'd be no longer true.

Reply:
donee

Reply:
resolving.